UnityProject/Assets/Scripts/Gateway/WorldGateway.cs

```diff
@@ -0,0 +1,171 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Mirror;
+
+
+/// <summary>
+/// Connects two portals together, can be separate from the station gateway.
+/// </summary>
+public class WorldGateway : NetworkBehaviour
```

Comment:
What is the difference between this and StationGateway.cs?

Reply:
Inheritance done